src/components/cards/RelationshipsCard.jsx

```diff
@@ -1,106 +1,511 @@
+    loading: loadingRelationships,
+  } = useSiteSettings();
+  const {
+    mutate: postRelationship,
+    error: postRelationshipError,
+    loading: postRelationshipLoading,
+    clearError: clearPostRelationshipError,
+  } = usePostRelationship();
+
+  const {
+    mutate: deleteRelationship,
+    loading: deleteRelationshipLoading,
+    error: deleteRelationshipError,
+    clearError: clearDeleteRelationshipError,
+  } = useDeleteRelationships();
+
   const theme = useTheme();
-  const backgroundColor = theme.palette.grey['200'];
-  return (
+  const [confirmDeleteDialog, setConfirmDeleteDialog] = useState({
+    relationshipGuid: null,
+    open: false,
+  });
+  const [
+    openRelationshipDialog,
+    setOpenRelationshipDialog,
+  ] = useState(false);
+  const [selectedIndividualId, setSelectedIndividualId] = useState(
+    null,
+  );
+  const [
+    selectedIndividualFirstName,
+    setSelectedIndividualFirstName,
+  ] = useState(null);
+  const [currentRoles, setCurrentRoles] = useState(null);
+  const [currentType, setCurrentType] = useState(null);
+  const [currentRole1, setCurrentRole1] = useState(null);
+  const [currentRole2, setCurrentRole2] = useState(null);
+  const allValid =
+    selectedIndividualId &&
+    currentType &&
+    currentRole1 &&
+    currentRole2;
+  const types = useMemo(
+    () => {
+      const possibleRelationships = get(
+        siteSettings,
+        ['relationship_type_roles', 'value'],
+        [],
+      );
+      const _types = Object.values(possibleRelationships);
+      setCurrentRoles(get(_types, 'roles', []));
+      return _types;
+    },
+    [siteSettings],
+  );
+  const relationshipTableData = useMemo(
+    () => {
+      return map(
+        relationships,
+        relationship => {
+          // keeping this in until we after QA and confirm for certain that this would not be needed (I have a note in the ticket to come back and clean this up)
+          // const selfIndividualMembers = filter(
+          //   get(relationship, 'individual_members'),
+          //   individualMember =>
+          //     individualMember.individual_guid !== individualGuid,
+          //   [],
+          // );
+          // const selfIndividualMember = get(selfIndividualMembers, [
+          //   0,
+          // ]);
+          const nonSelfIndividualMembers = filter(
+            get(relationship, 'individual_members'),
+            individualMember =>
+              individualMember.individual_guid !== individualGuid,
+            [],
+          );
+          const nonSelfIndividualMember = get(
+            nonSelfIndividualMembers,
+            [0],
+          );
+          return {
+            guid: relationship?.guid,
+            nonSelfFirstName:
+              nonSelfIndividualMember?.individual_first_name,
+            nonSelfGuid: nonSelfIndividualMember?.individual_guid,
+            type: relationship?.type_label,
+            role: nonSelfIndividualMember?.individual_role_label,
```

Comment:
This is the part that I found strange: the role from selfIndividualMember's perspective should be selfIndividualMember.individual_role_label, but it is not; it's the opposite. 
Which I suppose is fine as long as the correct thing is being displayed (which it is)... but I suspect that the roles are being swapped on the BE or something (or maybe in my API call).

In any case, not really a huge problem, just something that's likely to be confusing in the future.